ned_frame_origin:
  latitude: 0                       # [°]
  longitude: 0                      # [°]
  depth: 0                          # [m]
